scripts/test-resolve-native-cli-release-target.sh

```
@@ -263,49 +224,73 @@ run_failure_case() {
+    cd "$work_dir"
+    write_manifest 3.0.0-beta.3
+    PATH="$mock_bin:$ORIGINAL_PATH" \
+      CURRENT_VERSION=3.0.0-beta.3 \
+      CURRENT_RELEASE_STATE=missing \
+      CURRENT_RELEASE_HAS_ASSETS=false \
+      STAMP_KIND=none \
+      BUILD_SHA_VALUE=target-sha \
+      EVENT_NAME=push \
+      EVENT_REF_NAME=v3-beta \
+      INPUT_RELEASE_TAG= \
+      INPUT_DRY_RUN=false \
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)